src/content/manifest/hybrid-development.mdx

```diff
@@ -0,0 +1,59 @@
+
+> Using `hybrid` development mode breaks the replicability between operating systems and increases latency when accessing the rest of the services in the cluster. 
+
+### How hybrid development works
+
+To allow communication to and from the cluster and your local machine an SSH server is deployed in the remote development container. You can expose the local service to the development container using the [`reverse`](/reference/okteto-manifest.mdx/#reverse-string-optional) configuration for the development environment. If you want to forward a service exposed in the cluster to your local machine you can use the [`forward`](/reference/okteto-manifest.mdx/#forward-string-optional) configuration.
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/hybrid-development-arch.png").default}
+    alt="Hybrid development architecture"
+    width="850"
+  />
+</p>
+
+When using `hybrid` mode the process running locally is designed to behave similarly as if it were running in `sync` mode. To achieve this the following environment variables are added to the local process running the service in `hybrid` mode. This list is ordered from highest to lowest prevalence:
+- Environment variables declared in the `dev` section of the manifest
+- Environment variables of the remote development container
+- Environment variables of the configmap associated with the dev environment
+- User-defined variables
+- Environment variables declared in the image of the remote development container
+
+### How to develop in hybrid mode
+
+To enable `hybrid` mode you only need to configure the necessary ports to expose the local service to the remote development container. This allows the components running in the cluster to communicate with the local service as if it was running directly in the cluster. 
+
+To do this you can make use of the [reverse](reference/manifest/#reverse-string-optional) field in your Okteto manifest, which will expose the configured port of your local machine to the desired port on the development container.
+
+```yaml
+dev:
+  frontend:
+    mode: hybrid
+    reverse: 
+    - 3000:8080
+    workdir: ./frontend
+    command: ["yarn start"]
+```
+
+In this example we have configured the `frontend` service to use `mode: hybrid`. This means Okteto will redirect requests to the frontend service to your local machine and not in the development container.
+
+Okteto will execute (locally) the command defined in the `command` field and in the directory specified in `workdir`. Additionally, anything exposed locally on port `8080` will be exposed on port `3000` of the development container. This makes it possible for other components running on the cluster to access the local service as if it were running remotely in the cluster.
```

Comment:
`Okteto will execute (locally) the command defined in the `command` field and in the directory specified in `workdir` -> I would mention here that we also autoinject environment variables to simulate the same environment than the original container. With a link to where this is documented

Reply:
a52b4aea3ea4ea89f11c21d64b43d6526b200670